ping/rust/Cargo.toml

```diff
@@ -31,5 +31,5 @@ libp2pv0490 = { package = "libp2p", version = "0.49.0", features = ["websocket",
 [dependencies."libp2pv0500"]
 package = "libp2p"
 git = "https://github.com/libp2p/rust-libp2p"
-rev = "368705a1465c4322948d14ee46d42475c472ca1e" # This should usually point to a commit on master.
+tag = "v0.50.0"
```

Comment:
I don't think moving to `tag` would work here, given that we use `sed` in our Dockerfile matching on the string `rev`.

I might have been not very clear. Sorry. Let's use the commit hash that `v0.50.0` points at.

Reply:
Oh I see

Reply:
Now that we have released `v0.50.0` we should actually add that as a release and have this point to the most recent hash on master here!

Reply:
Should this be part of this PR?

Reply:
It doesn't have to be. I think we can merge this separately ;)

Reply:
I opened https://github.com/libp2p/test-plans/pull/87 and also fixed the deprecation warnings there.